### PR TITLE
Adding a script to test sqlsrv connection pooling on Linux and Mac

### PR DESCRIPTION
**Pull request by ulvii**

### CODE REVIEW

File: test/sqlsrv/isPooled.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<?php
+include_once 'autonomous_setup.php';
+$conn1 = sqlsrv_connect( $serverName, $connectionInfo);
+$connId1 = ConnectionID($conn1);
+$conn1 = null;
+
```
Comment (yitam) [inline test/sqlsrv/isPooled.php]: any reason you set $conn1 to null?
did you mean to use sqlsrv_close()?
same with stmt in ConnectionID() below... sqlsrv_free_stmt() should be used

File: test/sqlsrv/sqlsrv_ConnPool_Unix.phpt [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+file_put_contents($odbcinst_ini, $current);
+
+//Creating a new php process, because for changes in odbcinst.ini file to affect pooling, drivers must be reloaded.
+print_r(shell_exec("php ./test/sqlsrv/isPooled.php"));
+
+//disable pooling by modifying the odbcinst.ini file
+$current = file_get_contents($odbcinst_ini);
+$current = str_replace("CPTimeout=5\n[ODBC]\nPooling=Yes\n",'',$current);
+file_put_contents($odbcinst_ini, $current);
+
```
Comment (yitam) [inline test/sqlsrv/sqlsrv_ConnPool_Unix.phpt]: would you save the original content of odbcinst.ini and use it to replace the modified content?

File: test/sqlsrv/isPooled.php (hunk 1 above)
Comment (ulvii) [inline test/sqlsrv/isPooled.php]: in fact, does it matter if you close first connection or not in your test?
- That's how you pool connections.

File: test/sqlsrv/sqlsrv_ConnPool_Unix.phpt (hunk 2 above)
Comment (ulvii) [inline test/sqlsrv/sqlsrv_ConnPool_Unix.phpt]: I think this is simple enough :)

**Pull request merged**